Upstream status: https://github.com/CCExtractor/ccextractor/pull/1505

From 272e0e1410135c3a5de105cecce400c7b2a9ed5b Mon Sep 17 00:00:00 2001
From: Maxim Cournoyer <maxim.cournoyer@gmail.com>
Date: Fri, 17 Mar 2023 09:49:29 -0400
Subject: [PATCH] linux/Makefile.am: Add missing generated header.

This header is generated by the pre-build.sh script.  The compilation
fails if it is missing.

* linux/Makefile.am (ccextractor_SOURCES): Add
../src/lib_ccx/compile_info_real.h.
---
 linux/Makefile.am | 1 +
 1 file changed, 1 insertion(+)

diff --git a/linux/Makefile.am b/linux/Makefile.am
index fddc26a5..f83f1eff 100644
--- a/linux/Makefile.am
+++ b/linux/Makefile.am
@@ -118,6 +118,7 @@ ccextractor_SOURCES = \
 				../src/lib_ccx/bitstream.h \
 				../src/lib_ccx/ccx_common_option.c \
 				../src/lib_ccx/ccx_common_common.c \
+				../src/lib_ccx/compile_info_real.h \
 				../src/lib_ccx/utility.c \
 				../src/lib_ccx/activity.c \
 				../src/lib_ccx/asf_functions.c \

base-commit: cb496a711923c984251483776e652ca9c027513c
-- 
2.39.1

